counterparty-core/counterpartycore/test/conftest.py

```diff
@@ -123,7 +123,7 @@ def shouldCheckForMockProtocolChangesAtBlock(change_name):
 _generate_random_asset = util.generate_random_asset
 
 
-def generate_random_asset():
+def generate_random_asset(subasset_longname=None):
```

Comment:
Why are we adding this parameter and then not using it?

Reply:
it's a mock function so it should respect the signature event if we don't use the argument.